IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/enums/C2DMessageEnums.kt

```diff
@@ -1,6 +1,6 @@
 package com.iotconnectsdk.enums
 
-enum class C2DMessageEnums(val value: Int) {
+internal enum class C2DMessageEnums(val value: Int) {
```

Comment:
@sw-tt-rajkiranvank @sw-tt-tusharojha what will be default code if you don't receive from your list?

Reply:
Hi @sw-ot-deveshmevada 
I have put the else condition accordingly